src/snapshot_middleware/project.rs

```diff
@@ -1,4 +1,6 @@
-use std::{borrow::Cow, collections::HashMap, path::Path};
+use path_slash::PathExt;
+use relative_path::RelativePath;
+use std::{borrow::Cow, collections::HashMap, path::Path, path::PathBuf};
```

Comment:
```suggestion
use std::{borrow::Cow, collections::HashMap, path::Path, path::PathBuf};

use path_slash::PathExt;
use relative_path::RelativePath;
```

In this project, we generally prefer to group std imports away from imports from other crates.